daemon/daemon.go

```diff
@@ -973,6 +974,20 @@ func (daemon *Daemon) setHostConfig(container *Container, hostConfig *runconfig.
 
 	container.Lock()
 	defer container.Unlock()
+
+	var stat syscall.Stat_t
```

Comment:
Put everything from this line down to hostConfig.BlkioWeightDevice=.... into a function such as setBlkioWeightDevice() in daemon_unix & daemon_windows.go. Make the Windows implementation a no-op.

BTW Windows does support stat - it's abstracted in pkg\system. But irrelevant here as Windows doesn't support Blkio*